app/src/main/java/ch/sdp/vibester/model/SongList.kt

```diff
@@ -0,0 +1,86 @@
+package ch.sdp.vibester.model
+
+import org.json.JSONArray
+import org.json.JSONObject
+import java.lang.IllegalArgumentException
+

```

Comment:
Maybe adding a class doc ?

Reply:
What does a song List represent. Also not a List<Song>. Doc may be important

Reply:
The Javadoc looks good now